typescript-test-samples/schema-and-contract-testing/tests/contract-testing.test.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+import event_1_0_0 from '../events/event-1.0.0.json';
+import event_1_2_0 from '../events/event-1.2.0.json';
+import event_1_3_0 from '../events/event-1.3.0.json';
+
+// Sample business logic code - we expect to have a full address in the "address" string, e.g. "2 Park St, Sydney, NSW 2000, Australia"
+const isAddressValid = (event) => {
+  if (!event || !event.address) 
+    return false;
+
+  const addressWords = event.address.split(",");
+  if (addressWords.length != 4)
+    return false;
+
+  return true;
+}
+
+test('business logic in the producer has not changed', async () => {    
```

Comment:
We've been using `describe` blocks as the root container for tests to add context and help with grouping. Could we do so here as well?

Reply:
done